src/lib.rs

```diff
@@ -454,8 +455,9 @@ impl Mastodon {
 
             };
 
-            let mut headers = Headers::new();
-            headers.set(Authorization(Bearer { token: (*data.token).to_owned() }));
+            let mut headers = HeaderMap::new();
+            let auth = HeaderValue::from_str(&format!("Bearer {}", data.token));
+            headers.insert(reqwest::header::AUTHORIZATION, auth.unwrap());
```

Comment:
I generally prefer to not include the crate name if I have use statements at the top, can you replace this with `header::AUTHORIZATION`? You can replace the use statement at 67 with `use reqwest::{self, HeaderMap, HeaderValue};`.